src/hs_output.h

```diff
@@ -18,7 +18,7 @@ typedef struct hs_output
   FILE* fh;
   char* path;
   pthread_mutex_t lock;
-  size_t id;
```

Comment:
This should be replaced with hs_checkpoint (I missed it we reworking the checkpoints)